feedbackdb/scripts/db/create_feedback_user.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import *
```

Comment:
Are you sure about "lots" of repetition? ;). Some repetition for sure. feedback_user_exists and has_privileges duplicate some lines. I left it alone because I didn't think moving that into a separate function would be any more readable.  I certainly could make a separate function that could be called by both, if you really like.  Not sure what I'd call it...  Also,  this code has been lifted-and-shifted from opg-lpa repo BTW, it went through a previous PR review.  (It will be removed from opg-lpa repo once the next PR goes through)

Reply:
Fine, can leave as-is